### DISCUSSION TITLE
The qha free energies obtained by pure harmonicty and scph are different

### DISCUSSION THREAD

**Discussion by gangliu-github** [ttadano/alamode#195] [Q&A] [answered]
I have calculated free enegy by two methods: in one method, only 2nd force constant is used and the other is scph.
I find in the result file of scph, the data of QHA-like term are also listed. But I find the data of the two methods are a bit different. Why? In the scph, the same 2nd xml file is used.

**Comment by ttadano** [comment, marked as answer]
In the SCPH mode, the temperature-dependent phonon frequencies are used to compute the QHA-like term, while the true harmonic frequencies (temperature independent) are used in the QHA mode. That's why these two values are different.

**Reply by gangliu-github** [reply]
I get it. It means the phonon frequencies are different. 
Thanks!
